### PR TITLE
Add several initiatives

### PR DESCRIPTION
**Pull request by sergesp**
Add: 

* Aid For Ukraine ( https://donate.thedigital.gov.ua/ ), 

* Serhiy Prytula Charity Foundation, 

* World Central Kitchen (works in Kharkiv, among other places, 4 of their staff were injured due to shelling), 

* Feed the Zoo ( https://3oo.store/ ).

### CODE REVIEW

**Comment by Qizhen-Yang** [comment]
Unfortunately, many people think that things are black and white, that to be against Russia means to be with Ukraine. This idea is due to the propaganda in your country, although they are not necessarily wrong, there must be questionable or biased reports, and many people are brainwashed as a result. The Internet is still dominated by Europe and America, and you think you have all the news, but that's only what you think. Pathetic.

**Comment by Qizhen-Yang** [comment]
Ukraine had sent soldiers to Hong Kong in 2019 to support its independence.
Do you support that?

**Comment by sergesp** [comment]
Ukraine did not "sent" anyone to Hong Kong protests, especially "soldiers". Any citizen of Ukraine that came to Hong Kong to support the protests did so on their own volition as private citizen. 

(Btw, this repo and PR is about Ukraine, not China.)

**Comment by Qizhen-Yang** [comment]
There is a officer of azov battalion, he even posted that on twitter.
And the azov battalion is a legitimate military organization in Ukraine.
And I think a soldier should represent his country, not only himself.
HE WENT TO HONG KONG IN UNIFORM.
On 5/24/2022 ***@***.***> wrote：

Ukraine did not "sent" anyone to Hong Kong protests, especially "soldiers". Any citizen of Ukraine that came to Hong Kong to support the protests did so on their own volition as private citizen.

(Btw, this repo and PR is about Ukraine, not China.)

**Comment by Qizhen-Yang** [comment]
There is a officer of azov battalion, he even posted that on twitter.
And the azov battalion is a legitimate military organization in Ukraine.
And I think a soldier should represent his country, not only himself.
HE WENT TO HONG KONG IN UNIFORM.

**Comment by webknjaz** [comment]
Qizhen-Yang since you're just repeating the ruscist nazi propaganda (with zero original thoughts, may I add) which is specifically targeted at supporting genocides, I'm going to assume that you're either a bot or a ruscist (using the duck typing principle). If you want to educate yourself, use the Internet search. We won't tolerate any support of murderous rhetoric here. I urge you to stop. If you don't support Ukraine's right for survival and people not being murdered and raped, it's not a place for you.

File: docs/README.md [hunk 1]
```diff
@@ -35,6 +41,7 @@ Please opt-in for a monthly payment, if possible.
 - 💵 [**Nova Ukraine**](https://novaukraine.org/donate/) — provides humanitarian aid and raises awareness about Ukraine. [501(c)3, Tax ID 46-5335435]
 - 💵 [**Save the Children**](https://support.savethechildren.org/site/Donation2?df_id=5746&mfc_pref=T&5746.donation=form1) — helps children by providing food and shelter. [501(c)3, Tax ID 06-0726487]
 - 💵 [**UNICEF USA**](https://www.unicefusa.org/war-ukraine?form=FUNKBHMZQDQ) — provides access to safe water, nutrition, health care, education and protection. [501(c)3, Tax ID 13-1760110]
+- 💵 [**Feed the Zoo**](https://3oo.store/) — a private initiative that coollects links to Ukrainian zoos, animal funds or shelters that need help.
```
Comment (webknjaz) [inline docs/README.md:44]:
```suggestion
- 💵 [**Feed the Zoo**](https://3oo.store/) — a private initiative that collects links to Ukrainian zoos, animal funds or shelters that need help.
```

**Comment by Qizhen-Yang** [comment]
If Ukraine didn't have Nazi groups like The Azov Battalion, I would definitely support Ukraine.
But there is, and I hope The Azov Battalion can die asap.

**Comment by sergesp** [comment]
It seems that this PR is going nowhere, so I'll close it.

**Pull request closed without merging**